extensions/reactive-oracle-client/deployment/pom.xml

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-reactive-oracle-client-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-reactive-oracle-client-deployment</artifactId>
+
+    <name>Quarkus - Reactive Oracle Client - Deployment</name>
+
+    <properties>
+        <reactive-oracledb.url>vertx-reactive:oracle:thin:@localhost:1521:XE</reactive-oracledb.url>
+        <!-- See https://hub.docker.com/r/gvenzl/oracle-xe -->
+        <oracle.image>gvenzl/oracle-xe:21.3.0-slim</oracle.image>
```

Comment:
I think we are already using this image somewhere else so maybe we should centralize it in the `build-parent`? But that can probably be done in a follow-up PR.

/cc @Sanne

Reply:
Right - we use it in `integration-tests/jpa-oracle/pom.xml` as well.

+1 to follow up later, not really a blocker.

Reply:
Will do